java-checks/src/main/java/org/sonar/java/checks/EnumMapCheck.java

```diff
@@ -38,6 +42,7 @@
 @Rule(key = "S1640")
 public class EnumMapCheck extends BaseTreeVisitor implements JavaFileScanner {
   private JavaFileScannerContext context;
+  private static final MethodMatcher mapPut = MethodMatcher.create().typeDefinition("java.util.Map").name("put").withAnyParameters();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "java.util.Map" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)